.github/workflows/haskell-ci.yml

```diff
@@ -30,6 +30,12 @@ jobs:
     strategy:
       matrix:
         include:
+          - compiler: ghc-9.2.1
+            allow-failure: false
+            compilerKind: ghc
+            compilerVersion: 9.2.1
+            setup-method: ghcup
+            allow-failure: false
```

Comment:
Just adding these attributes won't work. The remaining script also need updating so it can handle them. That's why I suggested running `haskell-ci regenerate`.

Reply:
As I said, regenerate didn't work. Thanks for handling it.

Reply:
It did work for me. See https://github.com/sjakobi/newtype-generics/pull/21.

Maybe your `haskell-ci` version was too old?!

Reply:
I had installed it from Hackage but suppose that wasn't the right choice at the time.  From git does work, thanks for introducing me to haskell-ci.

Reply:
Ah, yes. I should have been clearer that it needs to be installed from a recent `master`.